lib/findAccessorMinMax.js

```diff
@@ -6,7 +6,7 @@ var ComponentDatatype = Cesium.ComponentDatatype;
 var defined = Cesium.defined;
 
 var getAccessorByteStride = require('./getAccessorByteStride');
-var getComponentReader = require('./getComponentReader.js');
+var getComponentReader = require('./getComponentReader');
```

Comment:
Ah, thanks for noticing this. I saw a couple other cases of this and pushed the changes.